static/js/containers/CollectionDetailPage.js

```diff
@@ -112,10 +123,22 @@ class CollectionDetailPage extends React.Component {
         </div>
         <div className="tools">
           {
-            collection.is_admin &&
-            <a onClick={ showDialog.bind(this, DIALOGS.NEW_COLLECTION) }>
-              <i className="material-icons">settings</i>
-            </a>
+            collection.is_admin && [
+              <DropboxChooser
```

Comment:
This chooser is getting incorrect urls for selected videos.  For example, selecting a particular video on master branch gets a source_url of `https://dl.dropboxusercontent.com/1/view/r07a4tunjwlqc97/jellie2s.mp4` but on this branch gets `https://www.dropbox.com/s/g5s28d1723b8kvr/jellie2s.mp4?dl=0`, which is an HTML page with the video file embedded in it. However for some reason the video status is not being changed to 'Upload failed' (I'll open another issue for that soon).

Reply:
PS This option will fix it: `linkType = 'direct'`